app/classifier/classifier.jsx

```diff
@@ -258,15 +258,20 @@ class Classifier extends React.Component {
     }
     const { already_seen, finished_workflow, retired, selection_state, user_has_finished_workflow, selected_at } = subject;
     const workflowTranslation = translations.strings.workflow[workflow.id];
+    const mergedInterventions = Object.assign(
+      {},
+      classification.metadata.interventions,
+      {
+        message: !!showIntervention,
```

Comment:
Not related to this PR, but should this be the message that was shown, rather than a boolean? I'm just wondering how the experiment tracks which messages are successful if we don't include the message text in the classification.

Reply:
I'm also trying to remember why I set this as a boolean in the first place, but coming up blank.

Reply:
I don't think we need to track the actual message text right now. The UUID is a proxy for this from our gateway system to the downstream data, i.e. for external users to track who saw what message.

Longer term i think we need to ensure we control the message content is from an internally approved list and then we can record the ID / TYPE of the message for lookup.

Reply:
> I'm also trying to remember why I set this as a boolean in the first place, but coming up blank.

I think maybe to record that there was a message for the user to see vs the actual message, to be combined with the `opt_in` attribute to know if they saw it.

Reply:
Sure, I'm just missing how an experiment tests which interventions were successful without either an ID/type or the message text in the classification.

Reply:
I would say that if this is recording whether or not the message was shown, then this should be `messageShown: !!showIntervention` for clarity.

Reply:
happy to make the change but noting that it is unrelated to this pr. I assume this won't break any downstream client scripts as they aren't using this attribute.

@eatyourgreens you ok with this requested change?